values.yaml

```diff
@@ -206,6 +206,34 @@ matrix:
   # Human readable reason for why the homeserver is blocked
   disabledMessage: ""
 
+  # Matrix user verification service
+  uvs:
+    enabled: false
+    image:
+      repository: "matrixdotorg/matrix-user-verification-service"
+      tag: "v3.0.0"
+      pullPolicy: IfNotPresent
+    replicaCount: 1
+    # TODO: Find way how to generate access token without user interaction.
+    # Provide here access token for Matrix Synapse API.
+    # This must be token from admin user, currently can be fetched manually from Riot/Element dashboard.
+    accessToken: ""
+    # Disable check for non private IP range of homeserver. E.g. set to `true` if your homeserver domain resolves to a private IP.
+    disableIpBlacklist: true
+    # Auth token to protect the API
+    # If not set, it will be generated by Helm. Any calls to the provided API endpoints
+    # need have the header "Authorization: Bearer <token>".
+    authToken: ""
```

Comment:
Let's hide this from values.yaml and just let helm generate it by default. I doubt anyone would need to set this themselves